src/ucp/core/ucp_worker.c

```diff
@@ -1677,6 +1682,7 @@ static void ucp_worker_print_used_tls(const ucp_ep_config_key_t *key,
                                !amo_emul ? "amo" : "amo_am", &strb);
     ucp_worker_add_feature_rsc(context, key, am_lanes_map, "am", &strb);
     ucp_worker_add_feature_rsc(context, key, stream_lanes_map, "stream", &strb);
+    ucp_worker_add_feature_rsc(context, key, ka_lanes_map, "keepalive", &strb);
```

Comment:
minor: maybe use `ka` to shorten the output?

Reply:
done